src/app/get-table-row-id.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, ElementRef, EventEmitter, HostListener, Output } from '@angular/core';
+
+@Directive({
+  selector: '[appGetTableRowId]'
+})
+export class GetTableRowIdDirective {
+
+  @Output('sendId') sendDataEmitter = new EventEmitter<string>();
+
+  constructor(private el: ElementRef) { }
+
+  @HostListener('click')
+  onClick() {
+    this.sendId();
+  }
+
+  sendId(): void {
+    // console.log(this.el.nativeElement.getAttribute('id'));
+    this.sendDataEmitter.emit(this.el.nativeElement.getAttribute('id'));
```

Comment:
poczytaj o Renderer2 tutaj https://angular.io/api/core/Renderer2 i zastanów się jak możesz go wykorzystać aby zrealizował to samo :)